vehicle/tesla.go

```diff
@@ -99,7 +99,7 @@ func NewTeslaFromConfig(other map[string]interface{}) (api.Vehicle, error) {
 	pc := request.NewClient(log)
 	pc.Transport = &transport.Decorator{
 		Decorator: transport.DecorateHeaders(map[string]string{
-			"Authorization": "Bearer " + cc.ProxyToken,
+			"X-Authorization": "Bearer " + cc.ProxyToken,
```

Comment:
Is the original Tesla Token in the `Authorization` still maintained?

Reply:
Vorhanden? Ja, guckst du logfile.

Reply:
Der `X-Authorization` wird korrekt gesetzt. Aber die Proxy Commands fehlt der reguläre `Authorization`.  Vmtl. weil hier kein `oauth2.Transport` eingebaut ist.